gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixUtils.java

```diff
@@ -146,6 +153,59 @@ static void waitJobInitialization(
     log.info("Work flow {} initialized", workFlowName);
   }
 
+  /**
+   * Inject in some additional properties
+   * @param jobProps job properties
+   * @param inputTags list of metadata tags
+   * @return
+   */
+  public static List<? extends Tag<?>> addAdditionalMetadataTags(Properties jobProps,
```

Comment:
nit: seems possible to name more precisely - maybe `initializeCoreMetadataTags` or `initBaseEventTags`?